src/backend/services/context.py

```diff
@@ -31,7 +33,8 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send):
         request = Request(scope)
         context.with_deployment_name(request.headers.get("Deployment-Name", ""))
 
-        if has_header_user_id(request):
+        ignore_path = any(request.url.path.startswith(p) for p in _ignore_paths)
+        if not ignore_path and has_header_user_id(request):
```

Comment:
It would be nice to find a better solution for this. Basically the scim routes are being authenticated by setting the header to : "Authorisation: ....". But the current code always assume there is a jwt token in the auth header and fails to parse.

Reply:
From beatrix:

```
Yeah, I think the optimal thing would be:
Try and get JWT token
success use as user_id
fails continue
Try and get user id from header
success use as user_id
fails continue
Is SCIM request
success use "scim" as user_id
fails continue
Use default user id
```

Reply:
okay managed to fix this nicely by checking if "bearer" is in header. We use "basic" for scim.